roles/deploy_flask_app/README.md

```diff
@@ -0,0 +1,94 @@
+deploy_flask_app
+==================
+
+A role to deploy a simple flask application with RDS db cluster as backend.
+
+Requirements
+------------
+
+AWS User Account with the following permission:
+
+* ec2:DescribeInstanceAttribute
+* ec2:DescribeInstanceStatus
+* ec2:DescribeInstances
+* ec2:RunInstances
+* ec2:CreateKeyPair
+* elb:CreateLoadBalancer
+* rds:DescribeDBInstances
+* rds:DescribeDBSubnetGroups
+* rds:ListTagsForResource
+
+Role Variables
+--------------
+
+## variables to create new hosts and groups in inventory of in memory playbook.
+
+* **region** (str): Region where the app is to be deployed.
```

Comment:
Shouldn't the name of the variables start with the role name prefix?

Reply:
Not needed. This variable is used in module defaults.  However, it has been renamed in the role under a different variable name  - https://github.com/redhat-cop/cloud.aws_ops/pull/85/files#diff-5f9b057d7420b9ceb4a68361a867a55ac83e25a7d90057a654f9ca3418631e22R4. By doing this, I am not disturbing the variable used in the playbook. Changing it seemed not necessary.